db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True, blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255, null=True, blank=True)
+    race = models.ForeignKey("Race", on_delete=models.CASCADE)
```

Comment:
Why did you choose CASCADE here? It means that if Race gets deleted this skill will be gone too, but what if it's also referenced by other Race?